New beamer filetype

manolomartinez opened the pull request.

manolomartinez commented:
Creates a new `beamer` filetype that simply clones `pandoc`. This is useful so as to have separate `pandoc#command#autoexec_command`, as per [this issue](https://github.com/vim-pandoc/vim-pandoc/issues/398).

fmoralesc commented on ftplugin/beamer.vim:
```diff
@@ -1 +1,11 @@
+
+" init vim-pandoc-after, if present {{{2
+try
+    call pandoc#after#Init()
```
This is needed to load the compatibility plugins from vim-pandoc-after, yes.

fmoralesc commented:
LGTM. A small note in the docs would make it perfect - but I don't think it's essential ;)

manolomartinez commented:
Let me get to it. I've never edited vim-style docs, but it shouldn't be too hard :)

manolomartinez commented:
I've changed the bit about `ftdetect` and added it to `plugin/pandoc.vim`, which is, I think, the right way to do it. I have also modified the docs accordingly.

fmoralesc commented:
LGTM. Gracias por la contribución!
